### PR TITLE
fix Markdown formatting to follow one-sentence-per-line in windows-full.md

### PR DESCRIPTION
**Pull request by alexandair**
Fix Markdown formatting to follow one-sentence-per-line recommendation.
That's the only change per @mirichmo's request.

### CODE REVIEW

File: docs/building/windows-full.md [hunk 1]
```diff
@@ -60,18 +54,18 @@ Build manually
...
 What can you do with the produced binaries?
 =========================================
 
-**Important**: "We don’t support production deployments of these binaries on any platform". For PowerShell .NET (aka: FullCLR PowerShell) our recommendation is to continue using the PowerShell .NET version already shipping in Windows Client and Windows Server.
+**Important**: "We don’t support production deployments of these binaries on any platform". 
+For PowerShell .NET (aka: FullCLR PowerShell) our recommendation is to continue using the PowerShell .NET version already shipping in Windows Client and Windows Server.
 
 The primary reason to build the PowerShell FullCLR binaries is to test backward compatibility, and interoperability between .NET and CoreCLR.
-It is also important to mention that some features like PowerShell Workflows are not currently available in the CoreCLR version.
+It is also important to mention that some features like PowerShell Workflows are not currently available in the CoreCLR version.
```
Comment (lzybkr) [inline docs/building/windows-full.md]: extra space at the eol

File: docs/building/windows-full.md [hunk 2]
```diff
@@ -82,10 +76,9 @@ Running Dev version of FullCLR PowerShell
 
 Running FullCLR version is not as simple as CoreCLR version.
 
-If you just run `./powershell.exe`, you will get a `powershell`
-process, but all the interesting DLLs (such as
-`System.Management.Automation.dll`) would be loaded from the Global
-Assembly Cache (GAC), not your output directory.
+If you just run `./powershell.exe`, you will get a `powershell` process, 
+but all the interesting DLLs (such as `System.Management.Automation.dll`) would be loaded 
+from the Global Assembly Cache (GAC), not your output directory.
```
Comment (lzybkr) [inline docs/building/windows-full.md]: by convention, shouldn't at least part of this line (up to the comma) be merged with the line above?

File: docs/building/windows-full.md (hunk 1 above)
Comment (alexandair) [inline docs/building/windows-full.md]: fixed.

File: docs/building/windows-full.md (hunk 2 above)
Comment (alexandair) [inline docs/building/windows-full.md]: fixed.

**Pull request merged**